### ISSUE TITLE
[SPIKE] Converting Blog Traffic - Newsletter Module for Blog Posts

### ISSUE THREAD

**Issue by Gbaldassano** [MozillaFoundation/foundation.mozilla.org#11027]
### Context:

Picking up from where this convo left off - Audrey and the Marketing team are very interested in getting a module added to blog posts that promotes signing up for our newsletter. 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaGtpIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--a6c326b0d9faf944355df7fc17c40d16d2023b2f/Screenshot%202023-08-21%20at%203.42.20%20PM.png)

What are the next steps here? Have designs already been created?

**Comment by kristinashu** [comment]
Linking to this thread from moco https://mozilla.slack.com/archives/C4FMZ6EDS/p1692040297076169 - Elise is always happy to chat about all things UX so we should touch base with her once we have a few options.

We should explore several different options, test a few (at least on usertesting.com) and see what resonates with our audience. 

Let's also ensure we are properly tracking the impact of the updates.

**Comment by kristinashu** [comment]
Request from Audrey can be found here https://mozilla-hub.atlassian.net/jira/servicedesk/projects/MOFOTP/queues/custom/250/MOFOTP-6.

@Gbaldassano design can work on this next sprint but in the meantime it would be helpful to have any data related to signups so we can better understand what is currently working and not working.

**Comment by Gbaldassano** [comment]
@kristinashu Logging some info from my conversation with Audrey last week about this:

- Confirmed that it will be a module (not a pop up) and will be on every blog post (as part of page type) for MVP
-One thing she flagged was that we should let GloPro know ahead of time that this new section will be added to every blog post so they're not caught offguard (which I can do)
-@mtdenton just curious - if this is turned on by default, is there any chance this can have a check box that can disable it in specific situations? Or does building as part of a page type prevent that?
-Confirmed copy will be the same across all modules - @kristinashu to help brainstorm copy, Audrey asked that we share a mock up with her of how it will look ahead of time.

**Comment by Gbaldassano** [comment]
Hey @kristinashu & @nancyt1 let me know if you can view this report: [Blog/Newsletter sign ups](https://analytics.google.com/analytics/web/#/analysis/p321347134/edit/B8QUoHkkRkKhYDxM2rK0ag)

Currently waiting on a more accurate way to pull this - but I think this should give a general idea of where newsletter sign ups are currently coming from (like header, footer, etc) for users overall and then for users who land on or visit a blog page in particular. 

For all users, looks like the footer drives the most signs ups.
Users who land on or visit a blog, looks like the 'call out box' is the highest, followed by the footer (which makes sense, since the call out box is only on blog posts, right?)

Let me know if I've missed anything but hoping this helps!

**Comment by kristinashu** [comment]
We've decided to go with the MVP option of a page template (text is not cms editable and will appear on all blog pages).

A few considerations that have come up are:
- Where on page: Mid page vs Staff choose where on page vs “Pop up” from bottom of page
- How many pages: Appears automatically on every page vs By default on every page but can remove vs Need to add to every page
- Copy and email list: Multiple different copy options and email list options vs Same copy everywhere and same email list 

@Gbaldassano for future reference, do you think devs should outline the all different approaches we've discussed (e.g. part of page template, snippet that can be added/removed form individual pages, cms block) in this ticket?

**Comment by kristinashu** [comment]
Summary from Gen:

Audrey, the Web team, and I have been discussing ways we can tweak our approach to this module that will still meet the needs of various Teams with a bit more flexibility. We're going to revise our planned implementation so that we build out this newsletter module as a 'block' (as opposed to part of the page template) that can get added to blog posts as needed and allow for the selection of different newsletter lists.

Option 1) Template Approach (Previous plan)
-  On all blog posts
-  Promoting MoFo list for MVP
-  Designated to middle of page/post
-  Localized
-  Tracking in GA
-  Source URL in SF

Option 2) Updated 'Block' Approach (New plan)
-   Add to blog posts as desired
-   Newsletter list can be designated by blog author (from MoFo, Mozfest, etc lists)
-   Flexible placement within blog content
-   Localized
-   Tracking in GA
-   Source URL in SF

This middle ground solution will still enable us to capture the maximum number of users landing on our most highly visited blog posts while also allowing for more long term flexibility as we start talking about what this should look like 2024. 

[New PRD is here.](https://docs.google.com/document/d/17VA5brmV3LIC4jhFLC_d78EwZ34-S4CKLXi7ZcOFR0M/edit#heading=h.e1hdva6khlbd)
